ntbs-service/Services/AuthorizationService.cs

```diff
@@ -164,18 +164,18 @@ private bool UserHasDirectRelationToNotification(Notification notification)
 
         private bool UserBelongsToTbService(string tbServiceCode)
         {
-            return _userPermissionsFilter.Type == UserType.NhsUser && _userPermissionsFilter.IncludedTBServiceCodes.Contains(tbServiceCode);
+            return _userPermissionsFilter.IncludedTBServiceCodes.Contains(tbServiceCode);
         }
 
         private bool UserBelongsToPhec(string phecCode)
         {
-            return _userPermissionsFilter.Type == UserType.PheUser && _userPermissionsFilter.IncludedPHECCodes.Contains(phecCode);
+            return _userPermissionsFilter.IncludedPHECCodes.Contains(phecCode);
         }
 
         private bool UserBelongsToResidencePhecOfNotification(Notification notification)
         {
             var phecCode = notification.PatientDetails.PostcodeLookup?.LocalAuthority?.LocalAuthorityToPHEC?.PHECCode;
-            return _userPermissionsFilter.Type == UserType.PheUser
+            return _userPermissionsFilter.IsInAtLeastOneRegion
                    && _userPermissionsFilter.IncludedPHECCodes.Contains(phecCode);
         }

```

Comment:
Conceptually, I wonder if these would be better being methods on the `UserPermissionsFilter` object?

Reply:
I've moved them